ansible/roles/de-deploy-service/tasks/swap-service.yaml

```diff
@@ -0,0 +1,44 @@
+## needs other_color and current_color variables passed in, as well as service_name
+
+- include: restart-service.yaml
+  vars:
+    restart_service_name: "{{service_name}}_{{other_color}}"
+
+- name: wait for {{other_color}} server to be up
+  uri: url=http://localhost:8500/v1/health/service/{{service_name_short}}?passing&tag={{environment_name}}-{{other_color}} method=GET return_content=yes
+  register: health_check
+  until: health_check.content != "[]"
+  retries: 25
+  delay: 2
```

Comment:
Why not give it a full minute or 2 to come up? So 30 retries with 2~4 seconds between each?
But 50 seconds is close enough, so I don't mind sticking with it.